rand_xorshift/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.1.0] - 2018-07-16
+- Initial release
```

Comment:
For consistency with the other RNGs, this should probably be "Pulled out of the Rand crate".

Reply:
Already done in #5. The ISAAC remains also already got fixed.